src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculation {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculation {
```
this class should be abstract